setup.py

```diff
@@ -10,6 +10,9 @@
       py_modules=['pytest_variables'],
       entry_points={'pytest11': ['variables = pytest_variables']},
       install_requires=['pytest>=2.4.2'],
+      extras_require = {
+          'hjson':  ['hjson'],
+          'yaml' :  ['PyYAML']},
```

Comment:
Nit: Remove the whitespace surrounding the `=`.
Nit: `:` should not be preceded by whitespace, and should only be followed by a single space.